Documentation/engineering/updating-ryujit.md

```diff
@@ -2,7 +2,7 @@
 
 Following steps are necessary to pick up a new version of RyuJIT code generation backend from CoreCLR:
 
-1. From the master branch of the CoreCLR repo, copy header files that are part of the contract with RyuJIT from `src\inc` on the CoreCLR side, to `src\JitInterface\src\ThunkGenerator` on the CoreRT side.
+1. From the master branch of the Runtime repo, copy header files that are part of the contract with RyuJIT from `src\coreclr\src\inc` on the CoreCLR side, to `src\JitInterface\src\ThunkGenerator` on the CoreRT side.
 2. Inspect the diffs
     1. If an enum was modified, port the change to the managed version of the enum manually.
     2. If a JitInterface method was added or changed, update `src\JitInterface\src\ThunkGenerator\ThunkInput.txt` and run the generation script next to the file to regenerate `CorInfoBase.cs` and `jitinterface.h`. Update the managed implementation of the method in `CorInfoImpl.cs` manually.
```

Comment:
The `Microsoft.NETCore.Jit` package is not built anymore. This will need to be retooled - you can add TODO note about it.

Reply:
(I was already tinkering with this last weekend but had to drop it before finish line because the weekend ended and my weekdays are too short for CoreRT work. Should have the NuGet update that consumes this from the Microsoft.NetCore.App.Runtime package on Saturday.)